code/game/gamemodes/miniantags/demons/slaughter demon/slaughter_demon.dm

```diff
@@ -0,0 +1,312 @@
+		if(L.stat == CONSCIOUS)
+			continue
+		var/obj/item/organ/internal/regenerative_core/legion/core = L.get_int_organ(/obj/item/organ/internal/regenerative_core/legion)
+		if(core)
+			core.remove(L)
+			qdel(core)
+		L.emote("scream")
+		L.adjustBruteLoss(1000)
+		devoured++
+		L.forceMove(src)
+		ADD_TRAIT(L, TRAIT_UNREVIVABLE, "demon")
+		consumed_mobs.Add(L)
+		if(ishuman(L))
+			var/mob/living/carbon/human/H = L
+			if(H.w_uniform && istype(H.w_uniform, /obj/item/clothing/under))
+				var/obj/item/clothing/under/U = H.w_uniform
+				U.sensor_mode = SENSOR_OFF
+		if(L.mind)
+			if(!charge_holder.bloodcharge || !slam_holder.bloodcharge)
+				charge_holder.bloodcharge = TRUE
+				slam_holder.bloodcharge = TRUE
+			mindsabsorbed++
+			maxHealth += 10
+			adjustHealth(-100)
+			L.forceMove(src)
+	while(mindsabsorbed % 4 == 0 && mindsabsorbed > 0)
+		mindsabsorbed -= 4
+		maxHealth += 50
+	playsound(src, feast_sound, 100, TRUE, -1)
+
+
+/mob/living/simple_animal/demon/slaughter_demon/proc/perform_phaseout()
+	icon_state = "daemon"
+	var/obj/effect/decal/foundblood = 0
+	for(var/obj/effect/decal/cleanable/C in channel_target)
+		if(C.can_bloodcrawl_in())
+			foundblood = C
+			break
+	if(!foundblood)
+		to_chat(src, "<span class='warning'>The blood you were channeling has been cleaned away!<span>")
+		channeling = FALSE
+		REMOVE_TRAIT(src, TRAIT_IMMOBILIZED, "channelingblood")
+		return
+	if(!in_range(channel_target, src))
+		to_chat(src, "The blood is far away!")
+		channeling = FALSE
+		REMOVE_TRAIT(src, TRAIT_IMMOBILIZED, "channelingblood")
+		return
+	phased = FALSE
+	channeling = FALSE
+	absorb()
+	REMOVE_TRAIT(src, TRAIT_IMMOBILIZED, "channelingblood")
+	var/obj/effect/dummy/slaughter_demon/holder = new /obj/effect/dummy/slaughter_demon(get_turf(src))
+	sink_animation(foundblood)
+	src.ExtinguishMob()
+	src.forceMove(holder)
+
+/mob/living/simple_animal/demon/slaughter_demon/proc/sink_animation(atom/A)
+	var/turf/mob_loc = get_turf(channel_target)
+	visible_message("<span class='danger'>[src] sinks into [A].</span>")
+	playsound(mob_loc, 'sound/misc/enter_blood.ogg', 100, TRUE, -1)
+	new /obj/effect/temp_visual/dir_setting/bloodcrawl(mob_loc, src.dir, "jaunt")
+
+/mob/living/simple_animal/demon/slaughter_demon/adjustHealth(amount, updating_health = TRUE)
+	to_chat(world, "Took [amount] Damage")
+	if(stat_amplification && amount > 0)
+		to_chat(world, "Subtracting [amount * stat_amplification]")
+		amount = amount - (amount * stat_amplification)
+		to_chat(world, "Suffered [amount] damage!")
+	..(amount, updating_health)
+
+/mob/living/simple_animal/demon/slaughter_demon/UnarmedAttack(atom/A)
+	if(istype(A, /mob/living/carbon))
+		if(stat_amplification > 0)
+			melee_damage_lower = round(melee_damage_lower + (melee_damage_lower * stat_amplification))
+			melee_damage_upper = round(melee_damage_upper + (melee_damage_upper * stat_amplification))
+			to_chat(world, "Modified melee damage is [melee_damage_lower]")
```

Comment:
Please remove any debug messages that may have been forgotten. I see a few `to_chat` messages being sent to `world`.